sql/moz-fx-data-shared-prod/org_mozilla_ios_firefox_derived/review_checker_events_v1/metadata.yaml

```diff
@@ -0,0 +1,20 @@
+friendly_name: Firefox iOS review checker events data
+description: |-
```

Comment:
Is it possible to add descriptions to these (even if it's the same description for each one)? I don't think it's self-evident what these tables are for, could be helpful to future users